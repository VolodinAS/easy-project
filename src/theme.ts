const theme = {
    color: {
        typeTask: '#F8BFBF',
        typeFeature: '#B7CFFF',
        typeBug: '#FEFBBD',
        black: '#000000',
        lightPurple: '#CFC7D9',
        purple: '#7A52AA',
        lightGray: '#F5F5F5',
        darkGray: '#9C9C9C',
        red: '#EB0C0C',
        white: '#FFFFFF',
        smallText: '#999999',
        opacityGray: '#ACACAC',
        landingColors: {
            mainGray: '#D5E6EA',
            mainBlue: '#3848A9',
            benefitsItemBackground: '#EDD8BE',
            buttonStartBackground: '#E43958',
            footerBackground: '#343F53',
            headerLogo: '#010101',
            headerBackground: '#F1E7DA'
        },
        dashboardColors: {
            overlayButtonCreate: '#ABE0E7',
            overlayButtonYes: '#B4E7AB',
            overlayButtonNo: '#F9CCBE',
        }
    }
};

export default theme;
