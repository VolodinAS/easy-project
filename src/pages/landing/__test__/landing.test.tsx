/* eslint-disable check-file/folder-naming-convention */
import React from 'react';
import { describe, it, expect, jest } from '@jest/globals';
import { render, fireEvent } from '@testing-library/react';
import { ThemeProvider } from '@emotion/react';
import { BrowserRouter } from 'react-router-dom';

import Landing from '../index';
import theme from '../../../theme';

describe('landing', () => {
    it('render', () => {
        render(
            <ThemeProvider theme={theme}>
                <BrowserRouter>
                    <Landing/>
                </BrowserRouter>
            </ ThemeProvider>
        );
        expect(document.body).toMatchSnapshot();
    });
});