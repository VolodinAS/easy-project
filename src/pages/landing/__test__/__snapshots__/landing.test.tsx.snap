// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing render 1`] = `
<body>
  <div>
    <div
      class="css-1c2k8sx"
    >
      <div
        class="css-17ot69y"
      >
        <div
          class="css-g035ff"
        >
          <div
            class="css-1c0ed4l"
          >
            <svg
              fill="none"
              height="70"
              viewBox="0 0 85 70"
              width="85"
              xmlns="http://www.w3.org/2000/svg"
            >
              <ellipse
                cx="9.94723"
                cy="14.3011"
                fill="#7A52AA"
                rx="9.63815"
                ry="9.03226"
              />
              <ellipse
                cx="42.8772"
                cy="9.03226"
                fill="#7A52AA"
                rx="9.63815"
                ry="9.03226"
              />
              <ellipse
                cx="42.4758"
                cy="42.5269"
                fill="#7A52AA"
                rx="18.8747"
                ry="17.6882"
              />
              <ellipse
                cx="75.0049"
                cy="14.3011"
                fill="#7A52AA"
                rx="9.63815"
                ry="9.03226"
              />
              <ellipse
                cx="75.0049"
                cy="48.1721"
                fill="#7A52AA"
                rx="9.63815"
                ry="9.03226"
              />
              <ellipse
                cx="9.94723"
                cy="48.1721"
                fill="#7A52AA"
                rx="9.63815"
                ry="9.03226"
              />
              <ellipse
                cx="42.8772"
                cy="68.4946"
                fill="#7A52AA"
                rx="9.63815"
                ry="1.50538"
              />
              <line
                stroke="#7A52AA"
                stroke-width="4"
                x1="10.3408"
                x2="10.3408"
                y1="19.5699"
                y2="34.6236"
              />
              <line
                stroke="#7A52AA"
                stroke-width="4"
                x1="75.3982"
                x2="75.3982"
                y1="19.5699"
                y2="34.6236"
              />
              <line
                stroke="#7A52AA"
                stroke-width="4"
                x1="75.0046"
                x2="8.34077"
                y1="33.6129"
                y2="33.6129"
              />
              <line
                stroke="#7A52AA"
                stroke-width="4"
                x1="43.2712"
                x2="43.2712"
                y1="16.5592"
                y2="68.4947"
              />
              <line
                stroke="#7A52AA"
                stroke-width="4"
                x1="75.0046"
                x2="8.34077"
                y1="48.6667"
                y2="48.6667"
              />
            </svg>
          </div>
          <div
            class="css-f16ntm"
          >
            EASY
            <br />
            PROJECT
          </div>
          <div
            class="css-1c0ed4l"
          >
            <div
              class="css-1kwb0gl"
            >
              <a
                class="css-rnyc8g"
                href="#product"
              >
                О продукте
              </a>
            </div>
            <div
              class="css-1kwb0gl"
            >
              <a
                class="css-rnyc8g"
                href="#intro"
              >
                Вступление
              </a>
            </div>
            <div
              class="css-1kwb0gl"
            >
              <a
                class="css-rnyc8g"
                href="#benefits"
              >
                Преимущества
              </a>
            </div>
            <div
              class="css-1kwb0gl"
            >
              <a
                class="css-rnyc8g"
                href="#uniq"
              >
                Уникальность
              </a>
            </div>
          </div>
          <section
            class="css-1prn45"
          >
            <button
              class="css-1640pth"
              title="language select"
              value="ru"
            >
              <img
                class="css-zlc08d"
                src="test-file-stub"
              />
            </button>
          </section>
        </div>
      </div>
      <div
        class="css-j0rufw"
        id="product"
      >
        <div
          class="css-1h99e8k"
        >
          <div
            class="css-8jrsb6"
          >
            <h2
              class="css-fyt07r"
            >
              Программное обеспечение для комплексного управления работой
            </h2>
          </div>
          <div
            class="css-wvrw00"
          >
            <p
              class="css-10dvk2y"
            >
              Программное обеспечение Easy Project предназначено для управления работой, оптимизировать проекты, упростить расчет цен, автоматизировать выставление счетов и оптимизировать менеджмент проектов.
            </p>
          </div>
          <a
            class="css-11g9kr1"
            href="/easy-project/registration"
            title="Регистрация"
          >
            <button
              class="css-1haqyjf"
            >
              <div
                class="css-1o6ajog"
              >
                Создать первый проект
              </div>
            </button>
          </a>
        </div>
        <div
          class="css-1c0ed4l"
        >
          <img
            alt="Логотип"
            class="css-gvwh4a"
            src="test-file-stub"
          />
        </div>
      </div>
    </div>
    <div
      class="css-1sfaa1j"
    >
      <div
        class="css-8j2qrm"
      >
        <div
          class="css-15pc35s"
        >
          <img
            alt="Логотип Иннополис"
            src="test-file-stub"
          />
        </div>
        <div
          class="css-15pc35s"
        >
          <img
            alt="Логотип RedCoder"
            src="test-file-stub"
          />
        </div>
      </div>
    </div>
    <div
      class="css-5j6wn1"
      id="intro"
    >
      <div
        class="css-1v38hm"
      >
        <h1
          class="css-1s264hv"
        >
          Готовы изменить то, как вы работаете?
        </h1>
      </div>
      <div
        class="css-12c6vpr"
      >
        <p
          class="css-1l5tvrx"
        >
          Вам не нужны десятки приложений, чтобы добиться цели. Узнайте, как переход на Easy Project позволит вам управлять всеми аспектами вашего бизнеса.
        </p>
      </div>
      <div
        class="css-18gglrl"
      >
        <iframe
          class="css-k3igyc"
          src="video-link"
        />
      </div>
      <a
        class="css-11g9kr1"
        href="/easy-project/registration"
        title="Регистрация"
      >
        <button
          class="css-1haqyjf"
        >
          <div
            class="css-1o6ajog"
          >
            Начать управление проектами
          </div>
        </button>
      </a>
    </div>
    <div
      class="css-efxc7m"
    >
      <div
        class="css-efxc7m"
      >
        <div
          class="css-1cxohti"
          id="benefits"
        >
          <h1
            class="css-1s264hv"
          >
            Преимущества Easy Project
          </h1>
        </div>
        <div
          class="css-1n76syx"
        >
          <div
            class="css-l8ch8q"
          >
            <div
              class="css-1muz9l"
            >
              <img
                alt="Бесплатно"
                class="css-1s10rmn"
                src="test-file-stub"
              />
            </div>
            <div
              class="css-t0lbh8"
            >
              <div
                class="css-1ak2gb8"
              >
                <p
                  class="css-dp2sq5"
                >
                  Абсолютно бесплатный
                </p>
              </div>
              <div
                class="css-g0ccg5"
              >
                <p
                  class="css-1l5tvrx"
                >
                  Весь доступный функционал навсегда в вашем распоряжении
                </p>
              </div>
            </div>
          </div>
          <div
            class="css-l8ch8q"
          >
            <div
              class="css-1muz9l"
            >
              <img
                alt="Оптимизация и быстрота"
                class="css-1s10rmn"
                src="test-file-stub"
              />
            </div>
            <div
              class="css-t0lbh8"
            >
              <div
                class="css-1ak2gb8"
              >
                <p
                  class="css-dp2sq5"
                >
                  Полная оптимизация
                </p>
              </div>
              <div
                class="css-g0ccg5"
              >
                <p
                  class="css-1l5tvrx"
                >
                  Не требует установки программ. Потянет даже на старом ведре
                </p>
              </div>
            </div>
          </div>
          <div
            class="css-l8ch8q"
          >
            <div
              class="css-1muz9l"
            >
              <img
                alt="Удобное использование"
                class="css-1s10rmn"
                src="test-file-stub"
              />
            </div>
            <div
              class="css-t0lbh8"
            >
              <div
                class="css-1ak2gb8"
              >
                <p
                  class="css-dp2sq5"
                >
                  Удобный интерфейс
                </p>
              </div>
              <div
                class="css-g0ccg5"
              >
                <p
                  class="css-1l5tvrx"
                >
                  Качественный, красочный и интуитивно понятный интерфейс
                </p>
              </div>
            </div>
          </div>
          <div
            class="css-l8ch8q"
          >
            <div
              class="css-1muz9l"
            >
              <img
                alt="Советские разработчики"
                class="css-1s10rmn"
                src="test-file-stub"
              />
            </div>
            <div
              class="css-t0lbh8"
            >
              <div
                class="css-1ak2gb8"
              >
                <p
                  class="css-dp2sq5"
                >
                  Советские разработчики
                </p>
              </div>
              <div
                class="css-g0ccg5"
              >
                <p
                  class="css-1l5tvrx"
                >
                  Сервис всегда будет работать стабильно, несмотря на любые жестокие санкции
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="css-ztsknp"
      >
        <div
          class="css-14eqxvj"
          id="uniq"
        >
          <h1
            class="css-1s264hv"
          >
            Что делает Easy Project уникальным?
          </h1>
        </div>
        <div
          class="css-1a0li2n"
        >
          <p
            class="css-1l5tvrx"
          >
            Тут красочное общее описание проекта
          </p>
        </div>
        <div
          class="css-lfvpx7"
        >
          <div
            class="css-dd0tq4"
          >
            <div
              class="css-1wwzegl"
            >
              01
            </div>
            <div
              class="css-4ljjmm"
            >
              <h3
                class="css-ftwof0"
              >
                Lorem ipsum dolor sit amet
              </h3>
            </div>
            <div
              class="css-gmgndr"
            >
              <p
                class="css-10dvk2y"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              </p>
            </div>
          </div>
          <img
            class="css-ipi10y"
            src="test-file-stub"
          />
        </div>
        <div
          class="css-1janjfa"
        >
          <div
            class="css-dd0tq4"
          >
            <div
              class="css-1wwzegl"
            >
              02
            </div>
            <div
              class="css-4ljjmm"
            >
              <h3
                class="css-ftwof0"
              >
                Lorem ipsum dolor sit amet
              </h3>
            </div>
            <div
              class="css-gmgndr"
            >
              <p
                class="css-10dvk2y"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              </p>
            </div>
          </div>
          <img
            class="css-ipi10y"
            src="test-file-stub"
          />
        </div>
        <div
          class="css-lfvpx7"
        >
          <div
            class="css-dd0tq4"
          >
            <div
              class="css-1wwzegl"
            >
              03
            </div>
            <div
              class="css-4ljjmm"
            >
              <h3
                class="css-ftwof0"
              >
                Lorem ipsum dolor sit amet
              </h3>
            </div>
            <div
              class="css-gmgndr"
            >
              <p
                class="css-10dvk2y"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              </p>
            </div>
          </div>
          <img
            class="css-ipi10y"
            src="test-file-stub"
          />
        </div>
        <div
          class="css-1dzmv67"
        >
          <a
            class="css-11g9kr1"
            href="/easy-project/registration"
            title="Регистрация"
          >
            <button
              class="css-1haqyjf"
            >
              <div
                class="css-1o6ajog"
              >
                Приступить к работе
              </div>
            </button>
          </a>
        </div>
      </div>
    </div>
    <div
      class="css-ewsonr"
    >
      <div
        class="css-eatzoa"
      >
        EASY PROJECT © 
        Все права защищены
      </div>
    </div>
  </div>
</body>
`;
